Core/Core/Groups/Group.swift

```diff
@@ -49,25 +52,9 @@ public final class Group: NSManagedObject, WriteableModel {
         model.name = item.name
         model.concluded = item.concluded
         model.showOnDashboard = !item.concluded
-        return model
-    }
-}
-
-extension Group {
-    public var color: UIColor {
-        let request = NSFetchRequest<ContextColor>(entityName: String(describing: ContextColor.self))
-        request.fetchLimit = 1
-        request.predicate = NSPredicate(format: "%K == %@", #keyPath(ContextColor.canvasContextID), canvasContextID)
-        if let color = try? managedObjectContext?.fetch(request).first {
-            return color.color
-        }
-        if let courseID = courseID {
```

Comment:
I don't know why we were overriding the color of the group with the color of the course. I don't think that's necessary so we don't do it anymore.

Reply:
I think if the group doesn't have a color, we should use the course color. It may not be a case that comes up anymore though.